x-pack/solutions/observability/plugins/apm/public/embeddable/trace_waterfall/trace_waterfall_embeddable.tsx

```diff
@@ -26,11 +24,11 @@ export function TraceWaterfallEmbeddable({
       return callApmApi('GET /internal/apm/unified_traces/{traceId}', {
         params: {
           path: { traceId },
-          query: { entryTransactionId, start: rangeFrom, end: rangeTo },
```

Comment:
q: why it's not needed anymore?

Reply:
First, this `entryTransactionId` was only defined but never used. Second, the unified trace waterfall doesn't need an entry document. It always construct the full waterfall searching for the root item (document without parent.id).